cli/cli
# Offer to add extra fields during `pr create` flow

This prototypes a flow during `gh pr create` that asks the user whether they would like to also select reviewers, assignees, labels, projects, and milestone. https://github.com/cli/cli/issues/340

This branch encapsulates "Option B" from our planning docs. "Option E" is found in the `pr-create-prototype-E` branch.

How to test:
- Check out this branch
- Run `make`
- Start `bin/gh pr create` - in this prototype, no PR will ever be submitted, so you are safe to choose "Submit"!

<img width="622" alt="Screen Shot 2020-04-09 at 8 20 13 PM" src="https://user-images.githubusercontent.com/887/78927666-8fd96800-7a9f-11ea-97ec-ced040b2cf1b.png">

Note: fake values (i.e. not read from the repository) are used as a list of available options.

To do:
- [x] seed the prototype with thousands of reviewer/assignee entries to check how it fares with large numbers
